Update architecture diagrams.

MancunianSam opened the pull request:
I've added in the Keycloak rotate secrets lambda and I've also removed
Jenkins and Grafana from the diagram on the same page.

TomJKing reviewed (commented):
First of all, wow, I'm impressed! ;) Just a couple of comments, the `tdr-beta-high-level-architecture` diagram is still showing Jenkins?

Also is it worth putting something in that diagram to show we now use Github actions instead of Jenkins?

MancunianSam commented:
I wouldn't have thought to do this but there was a Jira ticket for it :) I've removed the Jenkins bit from that diagram and added in GitHub actions outside the management account instead.

TomJKing reviewed (approved):
Looks good. Thanks
